contracts/cw20-base/src/allowances.rs

```diff
@@ -176,7 +179,8 @@ pub fn handle_burn_from(
         Ok(meta)
     })?;
 
-    let res = HandleResponse {
+    let res = Response {
```

Comment:
Would it make sense to extend `Response::default()`?

Or do something like:

```
let mut res = Response::default();
res.add_attribute("action", "burn_from");
res.add_attribute("from", owner);
res.add_attribute("by", deps.api.human_address(&spender_raw));
res.add_attribute("amount", amount);
Ok(res)
```

We could do this consistently if you like (at least on a few contracts as demo and make an issue to finish it).

NB: I would prefer the shorter (and more Ethereum like) `res.emit("from", owner)` but Simon didn't like that as it wasn't clear. Curious as to your opinion.

Reply:
I prefer it as it is now, for clarity.

A `default()` and overwriting is discouraged for style, by the way. See https://rust-lang.github.io/rust-clippy/master/#field_reassign_with_default 
 
A default can still be useful, though now that addresses don't have defaults, a "full default" is not possible anymore.